### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by nikhalim**
Added Chris Lian as an owner for easyRDMA

Signed-off-by: Kevin Lim <khai-wern.lim@ni.com>

### CODE REVIEW

**Comment by nikhalim** [comment]
FYI @chrislian-NI

File: CODEOWNERS
Comment (alejandro5042) [inline CODEOWNERS:2]: If you want to add `CHrislian-NI` to help with your review load, it has to be specified on the same line. Maybe this is what you intended?

From the docs: 

```
# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
# @global-owner1 and @global-owner2 will be requested for
# review when someone opens a pull request.
*       @global-owner1 @global-owner2
```

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

**Review by alejandro5042 — CHANGES_REQUESTED** [review]

File: CODEOWNERS
Comment (alejandro5042) [inline CODEOWNERS:2]: Also, I don't think they can see this repo?

image

**Review by alejandro5042 — CHANGES_REQUESTED** [review]

File: CODEOWNERS
Comment (nikhalim) [reply CODEOWNERS]: Hmm, I thought NI users would get access automatically. Can you help to grant the access to Chris? I don't think I have the permission to do so.

Comment (alejandro5042) [reply CODEOWNERS]: Great! Yes, Chris has to follow the [steps listed here](https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/6072/GitHub?anchor=how-do-i-get-access-to-ni-github-enterprise%3F).

Comment (alejandro5042) [reply CODEOWNERS]: I'll change my status to approve so I don't block the PR once Chris has access (when he can see this PR, for instance).

**Review by alejandro5042 — APPROVED** [review]

**Comment by CHrislian-NI** [comment]
Just managed to solve my GitHub account issue.

File: CODEOWNERS
Comment (nikhalim) [reply CODEOWNERS]: Chris has fixed his account issue, closing this thread.

**Pull request merged**
